AlertAnomalyRisk/jsonld-contexts/AlertAnomalyRisk.jsonld

```diff
@@ -4,10 +4,13 @@
     "Anomaly": "https://smartdatamodels.org/dataModel.AlertAnomalyRisk/Anomaly",
     "Risk": "https://smartdatamodels.org/dataModel.AlertAnomalyRisk/Risk",
     "conditions": "https://smartdatamodels.org/dataModel.AlertAnomalyRisk/conditions",
+    "diffusionList": "https://smartdatamodels.org/dataModel.AlertAnomalyRisk/diffusionList",
```

Comment:
looks a bit Frenchy?

Reply:
Actually a mailing list does not seem to especially refer to email so it's fine I guess